plugins/notifications-react/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-notifications-react",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "description": "Web library for the notifications plugin",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "web-library"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "workspace:^",
+    "@backstage/core-plugin-api": "workspace:^",
+    "@backstage/plugin-notifications-common": "workspace:^",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.11.3",
+    "lodash": "^4.17.21",
+    "material-ui-confirm": "^3.0.12",
+    "react-relative-time": "^0.0.9"
+  },
+  "peerDependencies": {
+    "react": "^17.0.0 || ^18.0.0"
```

```suggestion
    "@types/react": "^17.0.0 || ^18.0.0",
    "react": "^17.0.0 || ^18.0.0",
    "react-dom": "^17.0.0 || ^18.0.0",
    "react-router-dom": "^6.3.0"
```

That should get the CI happy 👍